pypeit/images/rawimage.py

```diff
@@ -669,8 +676,12 @@ def process(self, par, bpm=None, scattlight=None, flatimages=None, bias=None, sl
         # bias and dark subtraction) and before field flattening.  Also the
         # function checks that the slits exist if running the spatial flexure
         # correction, so no need to do it again here.
-        self.spat_flexure_shift = self.spatial_flexure_shift(slits, debug=debug) \
-                                    if self.par['spat_flexure_correct'] else None
+        self.spat_flexure_shift = None
+        if self.par['spat_flexure_method'] != "skip" or not np.ma.is_masked(manual_spat_flexure):
+            self.spat_flexure_shift = self.spatial_flexure_shift(slits, method=self.par['spat_flexure_method'],
```

Comment:
Hey @rcooke-ast . Just a little confusion here.
- `maxlag` is not a parameter of `self.spatial_flexure_shift()` anymore. `self.par['spat_flexure_maxlag']` is passed directly to `flexure.spat_flexure_shift()`
- you can do the same with `method`. Since it is defined in `self.par`, you can just pass it directly to `flexure.spat_flexure_shift()`

Reply:
Ah, yes, good call!  I'll update it 👍